test/built-ins/Iterator/zipKeyed/options-padding.js

```diff
@@ -0,0 +1,54 @@
+// Copyright (C) 2025 André Bargull. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-iterator.zipkeyed
+description: >
+  The "padding" option must be undefined or an object.
+info: |
+  Iterator.zipKeyed ( iterables [ , options ] )
+    ...
+    6. Let paddingOption be undefined.
+    7. If mode is "longest", then
+      a. Set paddingOption to ? Get(options, "padding").
+      b. If paddingOption is not undefined and paddingOption is not an Object, throw a TypeError exception.
+    ...
+features: [joint-iteration]
+---*/
+
+var validPadding = [
+  undefined,
+  [],
```

Comment:
Not a big deal, but this should probably be `{}` since `zipKeyed` doesn't rely on iterability.